### ISSUE TITLE
JMS resource recovery is erroneously processing a JNDI External Resource (ldap resource)

### ISSUE THREAD

**Issue by roberto-cisternino** [payara/Payara#2135]
In a new Payara 4.1.2.173 installation I created a standalone instance and configured an external JNDI resource (LDAP).  When I start the local instance without deploying any application I obtain the below reported log with a severe message.
The JMS resource recovery is erroneously processing the JNDI External Resource (ldap resource).

[2017-11-07T14:40:32.405+0100] [Payara 4.1] [INFO] [] [javax.enterprise.resource.resourceadapter.com.sun.enterprise.resource.recovery] [tid: _ThreadID=60 _ThreadName=Recovery Helper Thread] [timeMillis: 1510062032405] [levelValue: 800] [[JMS resource recovery has created CFs = 1]]

[2017-11-07T14:40:33.342+0100] [Payara 4.1] [SEVERE] [AS-JMS-CORE-00022] [javax.enterprise.resource.jms] [tid: _ThreadID=60 _ThreadName=Recovery Helper Thread] [timeMillis: 1510062033342] [levelValue: 1000] [[Error while loading connector resources during recovery : ldap/csi_ad]]

The external jndi resource has the following configuration that is fully working using GF 4.1.1:

    <external-jndi-resource jndi-lookup-name="dc=contship,dc=net" factory-class="com.sun.jndi.ldap.LdapCtxFactory" description="CSI Active Directory" res-type="javax.naming.ldap.LdapContext" jndi-name="ldap/csi_ad">
      <property name="java.naming.provider.url" value="ldap://10.5.1.49:3268"></property>
      <property name="java.naming.security.authentication" value="simple"></property>
      <property name="java.naming.security.principal" value="xxxxx@lsct.contship.net"></property>
      <property name="java.naming.security.credentials" value="xxxxx123"></property>
      <property name="java.naming.referral" value="follow"></property>
    </external-jndi-resource>
----------

## Expected Outcome

The external JNDI resource should be ignored by JMS

## Current Outcome

Then severity appears in the log but apparently this is not blocking other operations.
That LDAP external jndi resource can be later associated to a cluster or instance and works as expected.

## Steps to reproduce (Only for bug reports) 

1 -**  Start the defaul domain

    ./asadmin start-domain

2 -**  Setup a JNDI external resource or change directly the domain.xml before starting:

    <external-jndi-resource jndi-lookup-name="dc=contship,dc=net" factory-class="com.sun.jndi.ldap.LdapCtxFactory" description="CSI Active Directory" res-type="javax.naming.ldap.LdapContext" jndi-name="ldap/csi_ad">
      <property name="java.naming.provider.url" value="ldap://10.5.1.49:3268"></property>
      <property name="java.naming.security.authentication" value="simple"></property>
      <property name="java.naming.security.principal" value="xxxxx@lsct.contship.net"></property>
      <property name="java.naming.security.credentials" value="xxxxx123"></property>
      <property name="java.naming.referral" value="follow"></property>
    </external-jndi-resource>

3 -**  Create a new standalone instance XYZ

NOTE: DO NOT associate any resource to that new instance
(the upper JNDI external resource is by default associated to the server instance)

4 -**  Start the new standalone instance XYZ

    ./asadmin start-local-instance XYZ

In the log of the standalone instance you can read the error above reported:
"Error while loading connector resources during recovery : ldap/csi_ad"

## Context (Optional)

I am migrating a fully working WAR from GF 4.1.1

## Environment ##

- **Payara Version**: Payara Server 4.1.2.173 #badassfish (build 24)
- **Edition**: Full
- **JDK Version**: 1.8.0_152 Oracle
- **Operating System**: Windows 7 and others

**Comment by roberto-cisternino** [comment]
Same error is obtained using the previous release Payara Server Full 4.1.2.172

**Comment by roberto-cisternino** [comment]
This is a non-blocking issue with low priority (apparently)
